scripts/ecs_utils.py

```diff
@@ -138,7 +138,7 @@ def poll_cluster_state(ecs_client, cluster_name, service_names,
                         f'{service_name} tasks are still not healthy'
                     )
                     continue
-                if cluster_name in services[0]:
```

Comment:
Oh ok nevermind, I understand now, if the first service is prefixed with `cluster_name` then all of the services will be prefixed with `cluster_name`. I was pretty confused by that lol it took me 30 minutes figuring this out.

Could you do one of two things:
1. (Smaller change) Add an if statement at the beginning of this function along the lines of:
    ```
    # Starting in 2019, services are named <clustername>/servicename>
    is_2019_arn_format = services[0].startswith(f'{cluster_name}/')
    ```
    then change this `if` statement to `if is_2019_arn_format`. I think that is a bit more explicit.
2. (Cleaner but bigger change) Replace the `for loop` on line 128 to be something like
    ```
    for service_name, service_response in zip(services, response.get('services')):
    ```
    then you can remove this if statement altogether and just do
    `services.remove(service_name)`
    since `service_name` will come from the list of services directly so we don't have to reconstruct the ARN ourselves.

Reply:
Went for the smaller change. I'd prefer a cleaner approach, but I don't really understand this code and hesitant to do more than just fix the thing I need to unblock my other work.